src/main/java/Duke.java

```diff
@@ -1,10 +1,42 @@
+import duke.*;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+    private Storage storage;
+    private Tasklist tasks;
+    private Ui ui;
+
+    public Duke(String filepath) throws IOException {
```

Comment:
Perhaps the filepath variable should be named as filePath following the camel case convention?